### PR TITLE
promoting version 7.4.2-12

### PR DESCRIPTION
**Pull request by yuvallevy2**
Note - due to YAML handling changes done in the originating repo, there are many formatting changes that should be benign.

### CODE REVIEW

File: openshift/admission-service.yaml
Comment (zcahana) [inline openshift/admission-service.yaml:1]: Apparently this was missing from here the entire time?!

File: pvc_expansion.md
Comment (zcahana) [inline pvc_expansion.md:1]: There's an updated version of this readme merged to our internal repo, please update here as well.

Comment (zcahana) [reply pvc_expansion.md:1]: ^ Cancel that, we concluded that we'll omit this file from here and keep it internal.

File: openshift/admission-service.yaml
Comment (yuvallevy2) [reply openshift/admission-service.yaml:1]: Probably..

**Review by zcahana — APPROVED** [review]
LGTM

**Pull request merged**